docs/api/connection.md

```diff
@@ -107,12 +117,14 @@ To get the id of a connection and check if there is a connection with this id fo
     }
     ```
 
-### /api/users/:username/update/connection/token/:id `PUT`
-To update field data in connection with id. send data as a string with this fromat 
+### /api/users/:username/update/connection/data/:id `PUT`
+To update field token in connection with id.
 - Request
     ```json
-    {
-        "data" : {"data":{"host":"37.152.181.64","password":"sko192j3h","username":"reza"},"id":2,"name":"ssh1"}
+     "data": {
+        "host": "37.152.181.64",
+        "password": "-----------",
+        "username": "reza"
     }
     ```
```

Comment:
Open document of connection and compare it with other documents. Fix the issues. They should be JSON and wrapped by {}. Also cure about the tabs and spaces.

Reply:
https://github.com/Ragnacodes/Legato-Backend/blob/fix/v0.5/ssh/docs/api/connection.md

Reply:
done